libs/blocks/merch-card/merch-card.css

```diff
@@ -0,0 +1,376 @@
+  font-size: 12px;
+  font-weight: 700;
+}
+
+.merch-card .consonant-SpecialOffers-title h5,
+.merch-card .consonant-PlansCard-title h5,
+.merch-card .consonant-SegmentBlade-title h5 {
+  font-size: 12px;
+  color: #2C2C2C;
+  font-weight: 500;
+}
+
+.merch-card .consonant-ProductCard-prices,
+.merch-card .consonant-SegmentBlade-prices,
+.merch-card .consonant-SpecialOffers-prices,
+.merch-card .consonant-PlansCard-prices {
+  font-size: 16px;
+  line-height: 24px;
+  margin-bottom: 8px;
+  font-weight: 700;
+}
+
+.merch-card .consonant-ProductCard-description,
+.merch-card .consonant-SegmentBlade-description,
+.merch-card .consonant-SpecialOffers-description,
+.merch-card .consonant-PlansCard-description {
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 1.5;
+  margin-top: 0;
+}
+.merch-card .consonant-PlansCard-description {
+  margin-bottom: 18px
+}
+
+.merch-card .consonant-SegmentBlade-description span.placeholder-resolved,
+.merch-card .consonant-SpecialOffers-description span.placeholder-resolved,
+.merch-card .consonant-PlansCard-description span.placeholder-resolved {
+  font-size: 18px;
+  font-weight: 700;
+}
+
+.merch-card .consonant-SegmentBlade-description span.placeholder-resolved[data-template="priceStrikethrough"],
+.merch-card .consonant-SpecialOffers-description span.placeholder-resolved[data-template="priceStrikethrough"],
+.merch-card .consonant-PlansCard-description span.placeholder-resolved[data-template="priceStrikethrough"] {
+  font-size: 14px;
+  font-weight: 700;
+}
+
+.merch-card.consonant-SpecialOffers-Card {
+  border-radius: 16px;
+}
+
+.merch-card .consonant-SpecialOffers-img,
+.merch-card .consonant-PlansCard-img {
+  flex-grow: 1;
+  position: relative;
+  width: 100%;
+  min-height: 213px;
+  max-height: 213px;
+  background-color: #eaeaea;
+  background-position: 50% 50%;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.merch-card .consonant-SpecialOffers-ribbon,
+.merch-card .consonant-SegmentBlade-ribbon,
+.merch-card .consonant-PlansCard-ribbon {
+  position: absolute;
+  top: 16px;
+  right: 0;
+  font-size: 14px;
+  font-weight: 500;
+  max-width: 150px;
+  line-height: 16px;
+  text-align: center;
+  padding: 8px 11px 8px 11px;
+  border-radius: 5px 0 0 5px;
+}
+
+.merch-card .consonant-ProductCard-cell,
+.merch-card .consonant-SegmentBlade-cell,
+.merch-card .consonant-SpecialOffers-cell,
+.merch-card .consonant-PlansCard-cell {
+  justify-content: flex-end;
+  align-items: center;
+}
+
+p {
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 1.5;
+  margin-bottom: 8px;
+  margin-top: 0;
+  color: #000;
+}
+
+.merch-card h1,
+.merch-card h2,
+.merch-card h3,
+.merch-card h4,
+.merch-card h5,
+.merch-card h6 {
+  font-weight: 700;
+  line-height: 1.25;
+  margin-top: 0;
+  color: #000;
+}
+
+h2 {
+  font-size: 28px;
+  margin-bottom: 7px;
+}
+
+h3, h4 {
+  font-size: 20px;
+  margin-bottom: 8px;
+}
+
+h5 {
+  font-size: 18px;
+  margin-bottom: 4.5px;
+}
```

Comment:
Shouldn't these be scoped to a `.merch-card h5` etc. ?
Otherwise they will unintentionally affect other blocks header tags if these are used w/ other blocks or content. Also there are a bunch of css-linting issues i'm seeing on this file. Are you running a linter?

Reply:
done